src/dmd/mars.d

```diff
@@ -1706,6 +1706,7 @@ private bool parseCommandLine(const ref Strings arguments, const size_t argc, re
                             params.bug10378 = true;
                             break;
                         case 14488:
+                            printf("-transition=14488 is no longer needed.\n");
```

Comment:
Use `global.stdmsg`?

Reply:
I have seen `printf` being used here a couple of times, but sure `global.stdmsg` looks cleaner.